test/execution/test_plan.py

```diff
@@ -160,7 +160,7 @@ def serde_plan(x):
     serde_plan_detailed = serde._detail(bob, serde_plan_simplified)
 
     t = th.tensor([1.0])
-    expected = serde_plan_detailed(t)
+    expected = serde_plan(t)
```

Comment:
Wow, good catch! How did that happen?

Reply:
I think it's been that way since the test was written. Probably just a copy/paste oops.